packages/sitecore-jss-cli/src/micro-manifest.ts

```diff
@@ -110,3 +79,42 @@ export default async function microManifest(
     });
   });
 }
+
+/**
+ *
+ */
+export async function verifyArgs(argv: { [key: string]: any }) {
```

Comment:
What about ->
```suggestion
export async function verifyArgs(argv: { [key: string]: unknown }) {
```

Reply:
I think it should work, but I simply reused the signature from the main method for consistency:
```
export default async function microManifest(
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  argv: { [key: string]: any },
```